changelog.txt

```diff
@@ -6,6 +6,7 @@ Version 1.7.17
 * Fix function declaration without prototype on Xcode 14.3 (#1200)
 * Fix a crash when no additional info found in the device info json (#1203)
 * Add support PKeyAuthPlus and ECC based JWT signature generation. (#1044)
+* Created CIAM authority for MSAL (#1227)
```

Comment:
shouldn't this description need to go under TBD?

Reply:
updated to be under TBD